dateutil/test/test_isoparser.py

```diff
@@ -501,16 +516,16 @@ def test_isotime_midnight(isostr):
 ])
 def test_isotime_raises(isostr, exception):
     iparser = isoparser()
-    with pytest.raises(exception):
+    with pytest.raises(exception):              #pragma: no cover
         iparser.parse_isotime(isostr)
 
 
-@pytest.mark.xfail()
+@pytest.mark.xfail()                            #pragma: no cover
 @pytest.mark.parametrize('isostr,exception', [
     ('14:3015', ValueError),                    # Inconsistent separator use
     ('201202', ValueError)                      # Invalid ISO format
 ])
 def test_isotime_raises_xfail(isostr, exception):
```

Comment:
```suggestion
def test_isotime_raises_xfail(isostr, exception):    # pragma: nocover
```

This is the only line in this function that needs `# pragma; nocover`. The pragma goes after the function signature line on each of the `xfail` tests, and not in the body or on the decorator.